src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -198,13 +198,13 @@ export default App.extend(extend({
     });
   },
   onPostNewComment({ model }) {
-    model.set({ created_at: dayjs.utc().format() }).save();
+    model.set({ created_at: dayjs.utc().format() }).save().then(comment => {
```

Comment:
I think maybe just breaking up this line like:
model.set(..)
model.save().then

Also one question..  I'm not sure `then(comment` is the comment?  it's the result of the fetch I _think_.   But I'm not sure.. but `'ws', 'add', model` will always work within the `then`

Reply:
That makes sense. Updated to split them between lines.

Reply:
Ya it's a response stream. So to access that data we'd have to read it via `.json()`.

Like:

```js
model.save().then(response => {
  response.json().then(data => console.log(data))
});
```